Whisky/Views/Bottle/BottleView.swift

```diff
@@ -125,6 +142,19 @@ struct BottleView: View {
             }
             .disabled(!bottle.isActive)
             .navigationTitle(bottle.settings.name)
+            .toolbar {
```

Comment:
I'm not sure how I feel about the placement and icon of this button. `pin` to me doesn't really imply "Add a pin" and adding this would sort of clash with the bottle add button. We might need to rethink those buttons in general.

Reply:
I felt this way too a little bit. I would rather see a dummy icon with a plus with an add pin label below it rather than a toolbar button for this case. Then there is something to fill the list when there are not pins too.

Reply:
Something like this? It's using the toolbar text, but it can be whatever.

<img width="229" alt="Screenshot 2023-11-10 at 9 53 49 PM" src="https://github.com/Whisky-App/Whisky/assets/60248569/3ab6e154-4774-41c3-8f61-e83144d9def8">

<details><summary>Code to produce the above</summary>
<p>

It uses just enough of the `PinsView` layout to make me want to try to abstract it, but I'm honestly not sure it's worth it.

```swift
LazyVGrid(columns: gridLayout, alignment: .center) {
    ForEach(bottle.settings.pins, id: \.url) { pin in
        PinsView(
            bottle: bottle, pin: pin, path: $path
        )
    }
    VStack {
        Group {
            Image(systemName: "app.dashed")
                  .resizable()
                  .overlay {
                      Image(systemName: "plus")
                          .resizable()
                          .frame(width: 16, height: 16)
                  }
        }
        .frame(width: 45, height: 45)
        Spacer()
        Text("pin.help")
            .multilineTextAlignment(.center)
            .lineLimit(2, reservesSpace: true)
    }
    .frame(width: 90, height: 90)
    .padding(10)
    .onTapGesture(count: 1) {
        showPinCreation.toggle()
    }
}
.padding()
```
</p>
</details>

Reply:
This is in the latest commit.